worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -341,8 +373,13 @@ class PushDataHandler extends BaseMessageHandler with Logging {
     }
 
     // For test
-    if (conf.testPushDataTimeout && !PushDataHandler.pushDataTimeoutTested) {
-      PushDataHandler.pushDataTimeoutTested = true
+    if (isMaster && conf.testPushMasterDataTimeout && !PushDataHandler.pushMasterDataTimeoutTested) {
+      PushDataHandler.pushMasterDataTimeoutTested = true
```

Comment:
need to sleep for sometime to trigger timeout?

Reply:
> need to sleep for sometime to trigger timeout?

not need

Reply:
> > need to sleep for sometime to trigger timeout?
> 
> not need

Got it, the method is returned without reply, so timeout will be triggered.